lib/npm-install.js

```diff
@@ -19,19 +18,30 @@ function npmInstall(settings) {
     if (settings.stdio) {
       installArgs.stdio = settings.stdio;
     }
-  }
 
-  if (fs.existsSync('package-lock.json')) {
-    fs.removeSync('package-lock.json');
+    if (settings.stdio !== 'inherit') {
+      messageText += ' Add `--logLevel verbose` for realtime output.';
+    }
   }
 
+  const message = logger.promise(messageText);
   const npmProcess = spawn('npm', ['install'], installArgs);
 
   return new Promise((resolve, reject) => {
+    let error = '';
+
+    // This doesn't exist if we're logging verbosely
```

Comment:
By "this", do you mean the `stderr` property of the process?

Reply:
Yes.  I'll clarify the comment.